docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -639,6 +735,8 @@ Outputs a QR-code that represents URLs used to connect the [remote browsers](#re
 testcafe remote my-tests --qr-code
 ```
 
+*Related configuration file property*: [qrCode](configuration-file.md#qrcode).
+
 ### --sf, --stop-on-first-fail
 
 Stops an entire test run if any test fails. This allows you not to wait for all the tests included in the test task to finish and allows focusing on the first error.
```

Comment:
Stops a test run if any test fails.

Reply:
"This allows you not to wait for all the tests included in the test task to finish and allows focusing on the first error." allows allows?

Reply:
Oh, yes, I don't like this phrase at all, not just because of 'allows':

*Use this option when you are intended to fix failed tests one-by-one, and do not need a report on all the failures.*

Reply:
Use this option when you want to fix failed tests individually and do not need a report on all the failures.